src/admin/components/Fields/CustomChipField.tsx

```diff
@@ -24,7 +24,7 @@ const CustomChipField: React.FC<Omit<FunctionFieldProps, "render">> = props => (
           STATUS_CLASSNAME_MAP[record.readable_status] ?? "bg-grey-200 text-grey-500"
         )}
       >
-        {record.readable_status == "Started" ? "Draft" : record.readable_status}
+        {record.readable_status == "Unknown" ? "Started" : record.readable_status}
```

Comment:
Hmm, we shouldn't get an "Unknown" status from the server. That indicates something that needs fixing in the BE.